packages/venia-ui/lib/components/Accordion/section.js

```diff
@@ -0,0 +1,49 @@
+import React, { useCallback } from 'react';
+import { ChevronDown as ArrowDown, ChevronUp as ArrowUp } from 'react-feather';
+
+import { useAccordionContext } from './accordion';
+import Icon from '../Icon';
+
+import { mergeClasses } from '../../classify';
+import defaultClasses from './section.css';
+
+const Section = props => {
+    const { children, id, title } = props;
+
+    const {
+        handleSectionToggle,
+        lastSectionId,
+        openSections
+    } = useAccordionContext();
+    const isLast = lastSectionId === id;
```

Comment:
I apologize for the confusion caused by my last comment. It's fine to use pseudo-selectors such as `:last-child` or `:last-of-type`. I simply wanted to avoid using descendant selectors such as `.root > div` or `:root > *`. So I don't think we'll need `id` or `lastSectionId` logic. 😞

Reply:
No worries! 

`id` is still used to determine which `Section` is open, but I was able to nuke `lastSectionId` 👍

Reply:
I made that more clear in https://github.com/magento/pwa-studio/pull/2090/commits/82e01f193d85081e3dcde5dd870bb4284b6d43fa.